stock_picking_customer_ref/tests/test_stock_picking_customer_ref.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+# Copyright 2015 AvanzOSC - Alfredo de la Fuente
+# Copyright 2017 Tecnativa - David Vidal
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.tests import common
+
+
+class TestStockPickingCustomerRef(common.SavepointCase):
```

Comment:
Lets do open discussion. What is advantage using common.Savepoint Case than TransactionCase ?

Reply:
Thank you for the review. The difference between TransactionCase and SavepointCase is how it run the test. TransactionCase is run the test method independently (1 test method, 1 transaction), SavepointCase run all the test methods in one transaction. For example, we create 2 test methods. In TransactionCase, it will create 2 transaction, but in SavepointCase, it will create 1 transaction. So, the advantage is SavepointCase more faster than TransactionCase. CMIIW.